### DISCUSSION TITLE
Nextcloud addon database migration

### DISCUSSION THREAD

**Discussion by PhoeniXnr1** [alexbelgium/hassio-addons#1373] [Q&A]
I accidently choose the SQLite database instead of the MariaDB during the initial configuration of the Nextcloud Add-on, but I can't figure out how to migrate the database or restart the initial configuration again. I saw steps on how to do this in a normal deployment of Nextcloud, but this seems different for a home assistant / docker container installation. Could you please help me out?

**Comment by alexbelgium** [comment]
Hi, the only solution is to delete the addon and restart from scratch
